logstash-core/lib/logstash/pipeline.rb

```diff
@@ -211,17 +212,22 @@ def start_workers
   end
 
   # Main body of what a worker thread does
-  # Repeatedly takes batches off the queu, filters, then outputs them
+  # Repeatedly takes batches off the queue, filters, then outputs them
   def worker_loop(batch_size, batch_delay)
     running = true
 
+    namespace_events = metric.namespace([:stats, :events])
```

Comment:
I know that, just wondering what will be the expected output and behaviour for a metric namespace created, but without any data. 

I know now we create the namespace the same time we insert the data, so having no values stored means no namespace created. Do you think it could be the case we aim to show a metric with value zero?